packages/dataviews/src/types.ts

```diff
@@ -24,11 +24,17 @@ export interface Option< Value extends any = any > {
 	description?: string;
 }
 
-interface FilterByConfig {
+export interface OperatorConfig {
+	name: Operator;
+	label: string;
+}
+
+export interface FilterByConfig {
 	/**
 	 * The list of operators supported by the field.
+	 * Can be an array of operator keys, or an array of objects mapping operator keys to their config (e.g. label).
 	 */
-	operators?: Operator[];
+	operators?: ( OperatorConfig | Operator )[];
```

Comment:
Should this stay `Operator[]` then?

Reply:
I guess I'm not understanding what this FilterByConfig type applies to? is it the "Field type" or the "Field Type type"